### ISSUE TITLE
Proposal: Build a cli wallet on Kaia

### ISSUE THREAD

**Issue by FucktheKingcode** [kaiachain/kaia-dapp-mono#52]
Description
- Generates a keypair and import keypair
- Check balance of  cli wallet
- Handle faucet/sending Kaia

Bounty track
- Developer Tool

**Comment by zxstim** [comment]
I think you can try building something with [kaia-sdk](https://github.com/kaiachain/kaia-sdk)

**Comment by FucktheKingcode** [comment]
Yeah, thanks for sharing, I think I'll contribute an sdk in Rust to kaia-sdk. Something, maybe just a small part but I hope it helps

**Comment by zxstim** [comment]
I changed my mind now. I'm ok with a CLI wallet. Use kaia-sdk.
